.github/workflows/fast-forward.yml

```diff
@@ -9,19 +9,60 @@ jobs:
     name: Fast Forward
     if: |
       (github.event.issue.pull_request != '') &&
-      github.event.comment.body == '/fast-forward' &&
-      (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER')
+      github.event.comment.body == '/fast-forward'
     runs-on: ubuntu-latest
     steps:
+      - uses: actions-cool/check-user-permission@v2
+        id: has_permission
+        with:
+          require: 'write'
```

Comment:
Could we also use `'admin'` here? Or is this too strict?
The user `github.actor` is the user who wrote the comment, right?

Reply:
Could we also use 'admin' here? Or is this too strict?
-> For me no problem, currently all people which need to deploy are admin too

The user github.actor is the user who wrote the comment, right?
-> yes, my tests confirmed that